<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hello</class>
 <widget class="QWidget" name="Hello">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>486</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background:rgb(255, 255, 255);
	font-size:20px;
	font-family:Century Gothic,sans-serif;
}
QFrame{
background: blue;
}
QFrame{
	border:sold 10px rgba(0,0,0);
	background-image: url(I:/coding/C++project/final_checker_v1/pictureSource/postcard3.png);
}
QLineEdit{
	color: #8d98a1;
	background-color:#405361;
}
QLineEdit{
	color: #8d98a1;
	background-color:#405361;
	font-size:18px;
	border-style:outset;
	border-radius:10px;
}
QPushButton{
	background:#CDC8B1;
	border-style:outset;
	border-radius:15px;
 
}
QPushButton:pressed{
	background-color:#CDCDC1;
	border-style:inset;
}
QCheckBox{
	background:rgba(85,170,225,0);
	color:white;
}
QLabel{
	background:rgba(85,170,225,0);
	color:white;
	font-style: italic bold;
	font size:50px;
}

</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>811</width>
     <height>641</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>number of checkers</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="a_single_mode">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>257</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>A Single Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AI_mode">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>335</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>AI Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="network_mode">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>413</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Network Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Century Gothic&quot;;
color:black</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;Chinese&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;Checker !&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
